tests/roles/nova_adoption/tasks/main.yaml

```diff
@@ -0,0 +1,88 @@
+        template:
+          secret: osp-secret
+          apiServiceTemplate:
+            override:
+              service:
+                internal:
+                  metadata:
+                    annotations:
+                      metallb.universe.tf/address-pool: internalapi
+                      metallb.universe.tf/allow-shared-ip: internalapi
+                      metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+                  spec:
+                    type: LoadBalancer
+            customServiceConfig: |
+              [workarounds]
+              disable_compute_service_check_for_ffu=true
+          metadataServiceTemplate:
+            enabled: true # deploy single nova metadata on the top level
+            override:
+              service:
+                metadata:
+                  annotations:
+                    metallb.universe.tf/address-pool: internalapi
+                    metallb.universe.tf/allow-shared-ip: internalapi
+                    metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+                spec:
+                  type: LoadBalancer
+            customServiceConfig: |
+              [workarounds]
+              disable_compute_service_check_for_ffu=true
+          schedulerServiceTemplate:
+            customServiceConfig: |
+              [workarounds]
+              disable_compute_service_check_for_ffu=true
+          cellTemplates:
+            cell0:
+              conductorServiceTemplate:
+                customServiceConfig: |
+                  [workarounds]
+                  disable_compute_service_check_for_ffu=true
+            cell1:
+              metadataServiceTemplate:
+                enabled: false # enable here to run it in a cell instead
+                override:
+                    service:
+                      metadata:
+                        annotations:
+                          metallb.universe.tf/address-pool: internalapi
+                          metallb.universe.tf/allow-shared-ip: internalapi
+                          metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+                      spec:
+                        type: LoadBalancer
+                customServiceConfig: |
+                  [workarounds]
+                  disable_compute_service_check_for_ffu=true
+              conductorServiceTemplate:
+                customServiceConfig: |
+                  [workarounds]
+                  disable_compute_service_check_for_ffu=true
+    '
+
+- name: Wait for Nova control plane services' CRs to become ready
+  ansible.builtin.include_tasks:
+    file: wait.yaml
+
+# TODO(bogdando): implement podified FQDNs check for Nova endpoints
+- name: Check that Nova endpoints are defined and pointing to the podified FQDNs and that Nova API responds
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    alias openstack="oc exec -t openstackclient -- openstack"
+
+    ${BASH_ALIASES[openstack]} endpoint list | grep nova
```

Comment:
Does this check fails if nova is not in the list?

Reply:
yes, grep returns 1

Reply:
Cool